manifests/d/DaCosySheeep/FactDownloader/1.0/DaCosySheeep.FactDownloader.locale.en-GB.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: DaCosySheeep.FactDownloader
+PackageVersion: "1.0"
+PackageLocale: en-GB
+Publisher: DaCosySheeep

```

```suggestion
Publisher: Team Tobsta
```
This should match what is written in control panel

---

It should be DaCosySheeep in the control panel. At least it is on the future releases. Team Tobsta is the name of my discord server LOL.